lib/widgets/my_account/change_password_form.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:workoutpersonalizer_frontend/constants/styles.dart';
+import 'package:workoutpersonalizer_frontend/models/user.dart';
+import 'package:workoutpersonalizer_frontend/widgets/my_account/custom_form_field.dart';
+
+class ChangePasswordForm extends StatefulWidget {
+  const ChangePasswordForm({Key? key}) : super(key: key);
+
+  @override
+  State<ChangePasswordForm> createState() => _ChangePasswordFormState();
+}
+
+class _ChangePasswordFormState extends State<ChangePasswordForm> {
+  final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
+
+  // TODO: Use user from BE (WP-72)
+  User defaultUser =
+      User("123", "Rob", "Boss", "robboss123", "robboss123@gmail.com");
+
+  // TODO: Fix "No Overlay widget exists above EditableText" issue (WP-70)
+  @override
+  Widget build(BuildContext context) {
+    return Form(
+      key: _formKey,
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: <Widget>[
+          const Text("Old Password", style: secondaryHeadingStyle),
+          const CustomFormField(hintText: "Enter your old password"),
+          const Text("New Password", style: secondaryHeadingStyle),
+          const CustomFormField(hintText: "Enter your new password"),
+          const Text("Confirm New Password", style: secondaryHeadingStyle),
+          const CustomFormField(hintText: "Re-enter your new password"),
```

Comment:
Not sure if this is intentional, but the `CustomFormField`'s show my input, rather than showing stars / bullets.